src/components/Board.js

```diff
@@ -64,6 +64,11 @@ class Board extends React.Component {
       totalDistance,
       isLoading: false,
     }));
+    this.handleOnBlur = this.handleOnBlur.bind(this);
```

Comment:
This shouldn't be necessary.

Just define them using ES6-style `=>` and it's handled automatically:

```javascript
this.handleKeyUp = (event) => {
  ...
};
```